.flake8

```diff
@@ -4,3 +4,4 @@ exclude =
     __pycache__,
     build
 max-complexity = 10
+max-line-length = 120
```

Comment:
Is this what we've settled on? Almost all the other repositories are using the pep standard `79`,

Reply:
let's leave this as-is and fix in a subsequent PR, lots of changes needed.